src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -856,6 +886,565 @@ components:
+          example: '2023-06-19T09:39:44Z'
+        updatedAtPrison:
+          type: string
+          description: The identifier of the prison that the prisoner was resident at when this resource was updated.
+          example: 'BXI'
+      required:
+        - reference
+        - skills
+        - interests
+        - createdBy
+        - createdByDisplayName
+        - createdAt
+        - createdAtPrison
+        - updatedBy
+        - updatedByDisplayName
+        - updatedAt
+        - updatedAtPrison
+    FutureWorkInterestsResponse:
+      title: FutureWorkInterestsResponse
+      type: object
+      description: Represents a Prisoner's future work interests (if applicable).
+      properties:
+        reference:
+          type: string
+          format: uuid
+          description: A unique reference reference for this FutureWorkInterestsResponse.
+          example: 814ade0a-a3b2-46a3-862f-79211ba13f7b
+        interests:
+          type: array
+          description: One or more future work interests that the Prisoner has.
+          items:
+            $ref: '#/components/schemas/FutureWorkInterest'
+        createdBy:
+          type: string
+          description: The DPS username of the person who created this resource.
+          example: 'asmith_gen'
+        createdByDisplayName:
+          type: string
+          description: The display name of the person who created this resource.
+          example: 'Alex Smith'
+        createdAt:
+          type: string
+          format: date-time
+          description: An ISO-8601 timestamp representing when this resource was created.
+          example: '2023-06-19T09:39:44Z'
+        createdAtPrison:
+          type: string
+          description: The identifier of the prison that the prisoner was resident at when this resource was created.
+          example: 'BXI'
+        updatedBy:
+          type: string
+          description: The DPS username of the person who last updated this resource.
+          example: 'asmith_gen'
+        updatedByDisplayName:
+          type: string
+          description: The display name of the person who last updated this resource.
+          example: 'Alex Smith'
+        updatedAt:
+          type: string
+          format: date-time
+          description: An ISO-8601 timestamp representing when this resource was last updated. This will be the same as the created date if it has not yet been updated.
+          example: '2023-06-19T09:39:44Z'
+        updatedAtPrison:
+          type: string
+          description: The identifier of the prison that the prisoner was resident at when this resource was updated.
+          example: 'BXI'
+      required:
+        - reference
+        - interests
+        - createdBy
+        - createdByDisplayName
+        - createdAt
+        - createdAtPrison
+        - updatedBy
+        - updatedByDisplayName
+        - updatedAt
+        - updatedAtPrison
+
+    InductionSummaryListResponse:
```

Comment:
These two objects (`InductionSummaryListResponse` and `InductionSummaryResponse`) are identical to the existing CIAG objects (which are prefixed with `Ciag`). I don't see any reason to change them yet, but having separate objects allows us to change them independently.